ui/app/templates/allocations/allocation/index.hbs

```diff
@@ -65,15 +65,23 @@
               <td data-test-ports>
                 <ul>
                   {{#each row.model.resources.networks.firstObject.reservedPorts as |port|}}
-                    <li>
+                    <li data-test-port>
                       <strong>{{port.Label}}:</strong>
-                      <a href="http://{{row.model.allocation.node.address}}:{{port.Value}}" target="_blank">{{row.model.allocation.node.address}}:{{port.Value}}</a>
+                      {{#if row.model.allocation.node.address}}
+                        <a href="http://{{row.model.allocation.node.address}}:{{port.Value}}" target="_blank">{{row.model.allocation.node.address}}:{{port.Value}}</a>
+                      {{else}}
+                        ...
```

Comment:
Should this be in a span with some WAI-ARIA state on it to indicate it's loading or at least `aria-label="Loading address"` maybe?

Reply:
You're definitely right it should. I'll sweep all these loading states after 0.8 🚢s